### PR TITLE
Improve typos configuration

### PR DESCRIPTION
**Pull request by szepeviktor**
Make typos config more strict: miss less future misspellings.

- [ ] Public API changes documented in changelogs (optional)

### CODE REVIEW

File: .typos.toml [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-    "[1-9A-Za-z]{4}( [1-9A-Za-z]{4}){7,}",
+    "[1-9A-Za-z]{4}( [1-9A-Za-z]{4}){8,}",
     # some heuristics for base64 strings with no false matches found at the
     # time of writing.
-    "[A-Za-z0-9+=]{72,}",
-    "([A-Za-z0-9+=]|\\\\\\s\\*){72,}",
-    "[0-9+][A-Za-z0-9+]{30,}[a-z0-9+]",
-    "\\$[A-Z0-9+][A-Za-z0-9+]{6,}[a-z0-9+]",
-    "\\b[a-z0-9+/=][A-Za-z0-9+/=]{7,}[a-z0-9+/=][A-Z]\\b",
+    "[0-9A-Za-z+=]{38,}",
```
Comment (szepeviktor) [inline .typos.toml:16]: This makes all other base64-related regexps unnecessary.

File: .typos.toml [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
+    '"ed25519:[^"]+": "[^"]+"',
+    '"sender_key": "[^"]+"',
+    '"x": "[^"]{43}"',
+    "\\bmat ba\\b",
+    "\\bSomOlz-t3ABvRUweGuLCpHFKsI6yhGCoqPyS7o5gpfenizdfHLqq-l7kgyr7lSbW_mTSyYutby",
+    '\bfHAblQV"',
```
Comment (szepeviktor) [inline .typos.toml:22]: These cannot be pin-pointed.

**Review by bnjbvr — COMMENTED** [review]
Thanks for the patch, but I fail to understand what it improves. It will go into the hidden directories, which might be nice or annoying according to the developer who runs into it. It replaces 6 false positives with 5 false positives. Does it do something else that I'm not seeing?

**Comment by szepeviktor** [comment]
You are right.
It deserves a "close".

I thought those cryptographic values could be - somehow - ignored in a simple way.

**Pull request closed without merging**